src/v/kafka/server/tests/alter_config_test.cc

```diff
@@ -391,6 +404,10 @@ FIXTURE_TEST(
 
     auto second_group_describe_resp = describe_configs(
       test_tp, std::make_optional(second_group_config_properties));
+    vlog(
+      test_log.debug,
+      "first_group_describe_resp: {}",
```

Comment:
If you want to leave logging here

s/first/second

Reply:
thanks! fixed